resources/config/subscribers.yml

```diff
@@ -3,9 +3,15 @@ services:
     class: GrumPHP\Event\Subscriber\StashUnstagedChangesSubscriber
     arguments:
       - '@config'
-      - '@git.repository'
+      - '@GrumPHP\Git\GitRepository'
       - '@grumphp.io'
```

Comment:
Currently no plans for that. it's something that is happening gradually while we are improving stuff. 
We could create a PR to fix them all, but it might break extensions.